gulpfile.js

```diff
@@ -105,6 +105,22 @@ gulp.task('start-containers-dev', function(){
     }); 
 });
 
+gulp.task('start-containers-prod', function(){
+    
+    var adminP;
+    if (!fs.existsSync('./api/clients/Admin-browserify-bundle.js'))
```

Comment:
Arf, true, i had forgotten about that.

It's possible to do otherwise:
https://github.com/gulpjs/gulp/blob/master/docs/recipes/running-tasks-in-series.md

Let me check it real quick.